features/attached_commands.feature

```diff
@@ -31,13 +31,16 @@ Feature: Command behaviour when attached to an UA subscription
         And I run `sh -c "ls /var/log/ubuntu-advantage* | sort -d"` as non-root
         Then stdout matches regexp:
         """
+        /var/log/ubuntu-advantage-license-check.log
         /var/log/ubuntu-advantage.log
         /var/log/ubuntu-advantage-timer.log
         """
         When I run `logrotate --force /etc/logrotate.d/ubuntu-advantage-tools` with sudo
         And I run `sh -c "ls /var/log/ubuntu-advantage* | sort -d"` as non-root
+        # We don't rotate empty log files. That's why the license-check logs are not rotated
         Then stdout matches regexp:
         """
+        /var/log/ubuntu-advantage-license-check.log
```

Comment:
Should not we have a `.1` here as well?

Reply:
There is a comment above.
```
# We don't rotate empty log files. That's why the license-check logs are not rotated
```

Reply:
:man_facepalming: 
I'm blind or something